application/client.plugins/projects/serial/src/lib/views/dialog/components.ts

```diff
@@ -52,21 +55,46 @@ export class SidebarVerticalPortDialogComponent implements OnInit, OnDestroy {
             if (typeof message !== 'object' && message === null) {
                 return;
             }
+            if (message) {
+                this._onSpyState(message);
+            }
```

Comment:
this is not your 
```
if (typeof message !== 'object' && message === null) {
                return;
}
```
but it's a mistake. 
should be
```
if (typeof message !== 'object' || message === null) {
                return;
}
this._onSpyState(message);

```
after this correction you can remove checking of message